clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/security/EcsCredentialsInitializer.java

```diff
@@ -76,6 +84,20 @@ public ECSCredentialsConfig ecsCredentialsConfig() {
             CredentialsConfig ecsCopy = new CredentialsConfig();
             ecsCopy.setAccounts(Collections.singletonList(account));
 
+            // If no naming strategy is set at the account or provider
+            // level then the NamerRegistry will fallback to Frigga
+            String namingStrategy =
+                StringUtils.firstNonBlank(
+                    ecsAccount.getNamingStrategy(),
+                    ecsCredentialsConfig.getDefaultNamingStrategy());
+            if (namingStrategy != null) {
+              Namer<EcsResource> namer = namerRegistry.getNamingStrategy(namingStrategy);
+              NamerRegistry.lookup()
+                  .withProvider(EcsCloudProvider.ID)
+                  .withAccount(ecsAccount.getName())
+                  .setNamer(EcsResource.class, namer);
```

Comment:
Did you mean to store this and use it later?

Reply:
The `NamerRegistry` is essentially a singleton within the `clouddriver` context. This call is here to register the configure naming strategy for the given account.